InternetHospital.BusinessLogic/Services/UploadingService.cs

```diff
@@ -64,9 +62,9 @@ public async Task<User> UploadAvatar(IFormFile image, User user)
             }
 
             string pathFile = $"/{folderName}/{user.UserName}/{avatarFolder}/{fileName}";
-            _context.Users.Update(user);
-            user.AvatarURL = pathFile; 
-            await _context.SaveChangesAsync();
```

Comment:
await _context.SaveChangesAsync(); - must be here, because in Register method (SignupController) avatar will not save to DB

Reply:
If user already exists in our DB it should work